src/app/screens/connectors/ConnectCitadel/index.tsx

```diff
@@ -83,7 +83,7 @@ export default function ConnectCitadel() {
 
   return (
     <ConnectorForm
-      title="Connect to your Citadel Node"
+      title={<h1 className="mb-6 text-2xl font-bold dark:text-white">Connect to your <u><a href="https://runcitadel.space/">Citadel</a></u> Node</h1>}
```

Comment:
Instead of the html tag you can [use Tailwind to style the link](https://tailwindcss.com/docs/text-decoration). We already use this within the project.